roboteq_diff_driver/launch/driver.launch

```diff
@@ -16,19 +16,25 @@
     <param name="port" value="/dev/ttyACM0" />
     <!-- specify baud for roboteq controller -->
     <param name="baud" value="115200" />
-    <!-- specify whether to use open-loop motor speed control (as opposed to closed-loop)-->
-    <param name="open_loop" value="false" />
-    <!-- specify robot wheel circumference in meters  -->
-    <param name="wheel_circumference" value="0.3192" />
     <!-- specify robot track width in meters  -->
-    <param name="track_width" value="0.4318" />
+    <param name="track_width" value="1.25" />
```

Comment:
Wait nevermind that's not correct, it *is* the full width. I was just confused because we never actually use the `.4318` from this launch file, it gets set in from monsterduck